tutorials/frontend/from_tensorflow.py

```diff
@@ -0,0 +1,217 @@
+"""
+Compile Tensorflow Models
+=========================
+This article is an introductory tutorial to deploy tensorflow models with TVM.
+
+For us to begin with, tensorflow python module is required to be installed.
+
+Please refer to https://www.tensorflow.org/install
+"""
+
+# tvm, relay
+import tvm
+from tvm import relay
+
+# os and numpy
+import numpy as np
+import os.path
+
+# Tensorflow imports
+import tensorflow as tf
+from tensorflow.core.framework import graph_pb2
+from tensorflow.python.framework import dtypes
+from tensorflow.python.framework import tensor_util
```

Comment:
No need to import `graph_pb2`, `dtypes`, and `tensor_util`.  These are not used.